lib/reverse_characters.rb

```diff
@@ -1,2 +1,8 @@
 def reverse_characters(str)
+    old = str.chars()
+    y=""
+    for i in 0..str.length-1
+        y=y+old.pop()
```

Comment:
Great! In Ruby, we don't _have_ to include the `()`s when calling a method, unless we are passing in parameters.

Also, is there an operator that you can use as shorthand for this kind of operation: `some_var = some_var + something_else`?